client/src/components/admin/EditSurveyInfo.tsx

```diff
@@ -371,6 +425,7 @@ export default function EditSurveyInfo() {
         <div className={classes.actions}>
           <LoadingButton
             variant="contained"
+            disabled={!props.canEdit}
```

Comment:
[nit] I'd maybe rather hide the button (or the entire "actions" div) if the user cannot edit the survey... 🤔